features/support/admin_legacy_associations_helper.rb

```diff
@@ -0,0 +1,80 @@
+module AdminLegacyAssociationsHelper
+  def set_all_legacy_associations
+    tag_to_policy(policy_1)
+    tag_to_policy_area(policy_area_3)
+    tag_specialist_sectors
+    click_button "Save legacy associations"
+  end
+
+  def check_associations_have_been_saved
+    check_policies(policies: [policy_area_1, policy_1])
+    check_policy_areas
+    check_specialist_sectors
+  end
+
+  def check_legacy_associations_are_displayed_on_admin_page
+    assert_selected_policies_are_displayed
+    assert_selected_policy_areas_are_displayed
+    assert_selected_specialist_sectors_are_displayed
+  end
+
+  def stub_topics
+    @pol_area_1 = create(:topic, name: policy_area_1['title'])
+    @pol_area_3 = create(:topic, name: policy_area_3['title'])
+  end
+
+private
+
+  def assert_selected_policies_are_displayed
+    assert has_css? ".policies li", text: policy_1['title']
+    assert has_css? ".policies li", text: policy_area_1['title'] # TODO this seems wrong
```

Comment:
I can see that the test does `tag_to_policy_area(policy_area_3)`, I guess that's why the TODO is here?

Reply:
Again I think this is just the test setup, It does the right thing in reality.